
try {
let c = a + b
} catch (e) {
console.log(e)
} finally {
console.log('EXECUTOU')
}
